setup.sh

```diff
@@ -401,6 +401,11 @@ function safe_append()
   fi
 }
 
+function print_separator()
```

Comment:
Do we really need a function for this?

Reply:
Thinking about it, I don't think it's necessary.